docs/content/contributing/terraform/review.md

```diff
@@ -0,0 +1,75 @@
+
+{{< include file="static/includes/avm-terraform-module-review-template.md" language="markdown" options="linenos=false" >}}
+
+{{< /expand >}}
+
+7. The module reviewer can update the Compliance column for specs 42 to 47 to NA in case the module being reviewed isn't a pattern module.
+
+8. The module reviewer to review the code in the PR and make comments to request module owner of any necessary updates to the PR.
+
+9. The module reviewer will assign the AVM-Review issue to the module owner and respond to the module owner's comment on the Module Proposal in AVM Repo with the following
+
+{{< expand "➕ AVM Terraform Module Review Initiation Message" "expand/collapse" >}}
+
+"Thank you for reaching out to request a review of your module. The AVM module review process has been initiated, kindly perform the **Requested Actions** on the AVM-Review issue on the module repo."
+
+{{< /expand >}}
+
+10. The module owner will update the check list and the table in the AVM-Review issue and accordingly update the module reviewer.
+
+11. The module reviewer will perform the final review and ensure that all checks in the checklist are complete and the specs table has been updated with no spec's having compliance as 'No'.
+
+12. The module reviewer will approve the AVM-Review-PR and make the following comment on the AVM-Review issue and close it.
+
+{{< expand "➕ AVM Terraform Module Review Completion Message" "expand/collapse" >}}
+
+Thank you so much for contributing this module and completing the review process per AVM specs to ensure quality.
+
+You may proceed with publishing this module to the HashiCorp Terraform Registry with the initial pre-release version of v0.1.0.
+
+Please keep future versions also pre-release (e.g., 0.1.0, 0.1.1, 0.2.0, etc.) until AVM goes to GA of which the AVM team will notify.
+
+**Requested Action**: Once the module is published to the HashiCorp Terraform Registry kindly update the proposal with the below comment.
+
+"The AVM Review of this module has been completed and the module has been published to the registry. Requesting AVM team to close this module proposal and update module index accordingly."
+
+{{< /expand >}}
+
+13. The module owner will merge the AVM-Review-PR and publish the module to Terraform registry and update the module proposal on the AVM repo with the following
+
+{{< expand "➕ AVM Terraform Module Publish Message" "expand/collapse" >}}
+
+The AVM Review of this module has been completed and the module has been published to the registry. Requesting AVM team to close this module proposal and update module index accordingly.
+
+{{< /expand >}}
+
+14. The module reviewer will update the module proposal by doing the following
+- Move the module proposal from "In Review" to "Done" in the [AVM - Module Triage](https://github.com/orgs/Azure/projects/529) project.
+- Assign label Status: Module Available :green_circle:
+- Update the module index
+- Close the module proposal
```

Comment:
@mbilalamjad, where do we check if the mandatory GH teams have been created with the required permissions granted?

Reply:
Its one of the specs in the table which we ask the module owner to confirm even though we have automation to check it